### DISCUSSION TITLE
Password security

### DISCUSSION THREAD

**Discussion by khoavukhoa3004** [MauDucKG/BigFarm#18] [Q&A] [answered]
Khi lưu mật khẩu vào database, nhóm phát triển có mã hóa chưa ạ??

**Comment by duchuykg** [comment, marked as answer]
Việc sử dụng mã hóa mật khẩu chỉ là một phần trong việc bảo vệ cơ sở dữ liệu, nhóm vẫn đang hoàn thiện tính năng bảo vệ cơ sở dữ liệu.
